pipeline-examples/parallel-from-grep/README.md

```diff
@@ -14,10 +14,9 @@ circumstances.
 it's not really possible to use Groovy closures or syntax that depends
 on closures, so you can't do the Groovy standard of using
 .collectEntries on a list and generating the steps as values for the
-resulting entries. You also can't use the standard Java syntax for For
+resulting entries. You also can't use the standard Java syntax for "for"
```

Comment:
This would look better in backticks: `for`

Reply:
Good recommendation, @dresnick-sf. Wrapped some of the code related elements with backticks.

Reply:
It should be 
`
not
"